src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const first = (a) => {
+    count++;
+
+    const second = (b) => {
+      if (count < 3 || count % 2 !== 0) {
+        return a + b;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
that's what I meant on the QnA
```suggestion
      const shouldBreak = count > 3 || count % 2 === 0;
      return shouldBreak
        ? 'Bzzz... Error!'
        : a + b
```